mautic/mautic
# Segments filters dictionary improvements

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | N
| New feature? | Y
| Automated tests included? | N
| BC breaks? | N
| Deprecations? | N

#### Description:

This commit enables two events.
One for feeding the filters lists from event listeners.
Second event for filters evaluation.

#### Steps to test this PR:

This commit should have no influence on anything. It means that no visible testable change will be.
Check that filters in segment edit are still the same as before checking out this PR.
Check that filters in dynamic contents are the same.
Check dynamic content filters do work correctly.
